templates/certified/search-results.html

```diff
@@ -136,9 +136,9 @@ <h2 class="p-heading--4">
               </ul>
               <hr style="margin-bottom: 1rem;">
             </aside>
-            <div class="u-align--right">
+            <div class="u-align--right" style="padding-right: 0.5rem">
               <button class="p-button--neutral p-update-results u-no-margin--right" onclick="return clearFilters()">Clear</button>
-              <button class="p-button--positive p-update-results js-apply-filters" type="submit" disabled>Apply</button>
+              <button class="p-button--positive p-update-results js-apply-filters" type="submit" onclick="hideDrawerPageReload()" disabled>Apply</button>
```

Comment:
Shouldn't we execute this only when necessary? When the toggle actually shows?

Reply:
I've updated the function - let me know if this is what you meant